### PR TITLE
Bumping pyomo tag (6.6.2 prerelease)

### PR DESCRIPTION
**Pull request by jsiirola**
## Summary/Motivation:
This bumps the Pyomo tag in preparation for the August release.

## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def __getitem__(self, key):
     # Put abstract (non-versioned) deps here.
     # Concrete dependencies go in requirements[-dev].txt
     install_requires=[
-        "pyomo>=6.6.1",
+        "pyomo @ https://github.com/IDAES/pyomo/archive/6.6.2.idaes.2023.07.27.zip"
```
Comment (lbianchi-lbl) [inline setup.py:129]:
```suggestion
        "pyomo @ https://github.com/IDAES/pyomo/archive/6.6.2.idaes.2023.07.27.zip",
```

**Review by andrewlee94 — APPROVED** [review]
LGTM

**Comment by jsiirola** [comment]
@andrewlee94, the `expr.current` PR has been merged upstream.  Do we want to merge this and bump the version again in #1234, or just bump the version here and ignore the deprecation warnings for a little while?

**Comment by andrewlee94** [comment]
@jsiirola I was going to merge this first, not that it really matters. I was just teaching @strahlw how to create and push a tag and then update the tag in his PR.

**Review by eslickj — APPROVED** [review]

**Comment by andrewlee94** [comment]
Closing this as it is superseded by #1234

**Pull request closed without merging**